2-collectors/scala-stream-collector/src/main/scala/com.snowplowanalytics.snowplow.collectors/scalastream/sinks/KafkaSink.scala

```diff
@@ -51,20 +53,9 @@ class KafkaSink(config: CollectorConfig, inputType: InputType.InputType) extends
    */
   private def createProducer: KafkaProducer[String, Array[Byte]] = {
 
-    info(s"Create Kafka Producer to brokers: ${config.kafkaBrokers}")
-
-    val props = new Properties()
-    props.put("bootstrap.servers", config.kafkaBrokers)
-    props.put("acks", "all")
-    props.put("retries", "0")
-    props.put("batch.size", config.byteLimit.toString)
-    props.put("linger.ms", config.timeLimit.toString)
-    props.put("key.serializer", 
-      "org.apache.kafka.common.serialization.StringSerializer")
-    props.put("value.serializer", 
-      "org.apache.kafka.common.serialization.ByteArraySerializer")
```

Comment:
please keep your changes to a minimum and modify the configuration where it originally was, ie here